internal/framework/subproviders/morpheus/convert/convert.go

```diff
@@ -61,6 +62,50 @@ func BoolToType(b *bool) types.Bool {
 	return types.BoolValue(*b)
 }
 
+func StringToBool(s string) types.Bool {
+	switch strings.ToLower(s) {
+	case "on", "true", "yes":
+		return types.BoolValue(true)
+	case "off", "false", "no":
+		return types.BoolValue(false)
+	default:
+		return types.BoolValue(false)
```

Comment:
It would make these functions a bit more annoying to use. But I wonder if we should error/warn if we reach the `default` case as it might produce unexpected results